peekingduck/pipeline/nodes/draw/poses.py

```diff
@@ -44,14 +47,95 @@ class Node(AbstractNode):
+            or a list of BGR values.
+
+        keypoint_connect_color (:obj:`Union[List[int], str]`): **default = "champagne"** |br|
+            Color of the keypoints should either be a string in :ref:`color-palette`,
+            or a list of BGR values.
+
+        keypoint_dot_radius (:obj:`int`): **default = 5** |br|
+            Radius of the keypoints.
+
+    **Color Palette**
+
+    :ref:`color-palette` offers a wide range of default colors for the user to choose from.
+
+    .. _color-palette:
+
+    .. list-table:: PeekingDuck's Color Palette
+       :widths: 20 20
+       :header-rows: 1
+
+       * - Color Palette by Type
+         - Color Palette by Name
+       * - .. figure:: ../assets/api/color_map_by_type.png
+
+           PeekingDuck's Color Palette Sorted by Color Types [1]_.
+
+         - .. figure:: ../assets/api/color_map_by_name.png
+
+           PeekingDuck's Color Palette Sorted by Color Names [1]_.
+
+    .. rubric:: Footnotes
+
+    .. [1] Colors with asterisk indicates PeekingDuck's in-house colors.
     """
 
     def __init__(self, config: Dict[str, Any] = None, **kwargs: Any) -> None:
         super().__init__(config, node_path=__name__, **kwargs)
-        self.keypoint_dot_color = tuple(self.config["keypoint_dot_color"])
-        self.keypoint_connect_color = tuple(self.config["keypoint_connect_color"])
-        self.keypoint_text_color = tuple(self.config["keypoint_text_color"])
+        self._validate_configs()
+
+        self.keypoint_dot_color = get_color(self.config["keypoint_dot_color"])
+        self.keypoint_connect_color = get_color(self.config["keypoint_connect_color"])
+
+        self.pose = Pose(
+            self.keypoint_dot_color,
+            self.keypoint_connect_color,
+            self.keypoint_dot_radius,
+        )
+
+    def _validate_configs(self) -> None:
+        """Validates the config values."""
+        self._check_valid_color("keypoint_dot_color")
+        self._check_valid_color("keypoint_connect_color")
+        self.check_bounds("keypoint_dot_radius", "[0, +inf)")
+
+    def _check_valid_color(self, key: str) -> None:
+        """Checks that configuration value specified by `key` is a valid color.
+
+        Example:
+            >>> keypoint_dot_color = "blueee"
+            >>> self._check_valid_color("keypoint_dot_color")
+
+            This will raise a ValueError as "blueee" is not a valid color.
+
+            >>> keypoint_dot_color = [100, 100, 300]
+            >>> self._check_valid_color("keypoint_dot_color")
+
+            This will raise a ValueError as 300 is out of range.
+
+        Args:
+            key (str): The specified key.
+
+        Raises:
+            ValueError: If the configuration value is not either a valid color
+                in :ref:`color-palette`, or a list of BGR values, or if the BGR
+                values are out of range [0, 255].
+        """
+        valid_colors_type = set(COLOR_MAP.keys())
```

Comment:
Apply `sorted()` to this so the error message later reads better

Reply:
As discussed, will apply `sorted` in the error message instead.